applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -540,9 +540,32 @@ Syntax
   * ``0`` - Get
   * ``1`` - Set
 
-For a complete list of the supported SET ``<name>`` accepted parameters, see the `SETSOCKETOPT Service Spec Reference`_.
+* The ``<name>`` parameter can accept one of the following values:
 
-For a complete list of the supported GET ``<name>`` accepted parameters, see the `GETSOCKETOPT Service Spec Reference`_.
+  * ``2`` - ``SO_REUSEADDR`` (set-only).
+    ``<value>`` integer value ``0`` for disabling or ``1`` for enabling.
+  * ``20`` - ``SO_RCVTIMEO``.
+    ``<value>`` integer value. Receive timeout in seconds.
+  * ``21`` - ``SO_SNDTIMEO``.
+    ``<value>`` integer value. Send timeout in seconds.
+  * ``25`` - ``SO_BINDTODEVICE`` (set-only).
+    ``<value>`` integer value. Bind socket to specific PDP context ID.
+  * ``30`` - ``SO_SILENCE_ALL``.
```

Comment:
Note that some of these commands are not publicly documented beyond: https://github.com/nrfconnect/sdk-zephyr/blob/main/include/zephyr/net/socket_ncs.h

Reply:
See my other comment above.